androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/LauncherActivity.java

```diff
@@ -267,6 +272,26 @@ protected Uri getLaunchingUrl() {
         return TwaLauncher.CCT_FALLBACK_STRATEGY;
     }
 
+    protected TrustedWebActivityDisplayMode getDisplayMode() {
```

Comment:
I think this method should be in `LauncherActivityMetadata`. That class should do the parsing from Strings to domain types.

Reply:
Cutout mode can have 3 different values, while sticky has 2. That leads to 6 different possible values.  We can either cover all values as strings or specify what be believe are the most common options and let developers override if needed.

Reply:
changed.